scout/server/blueprints/institutes/templates/overview/utils.html

```diff
@@ -110,7 +110,7 @@ <h5 class="header">General Institute Settings</h5>
               <!-- Alamut settings -->
               <div class="col-md-3">
                 {{ form.alamut_key.label(class="control-label",data_toggle="tooltip", data_placement="top", title="Optional Alamut Visual Plus API key to be used when opening links to Alamut") }}
-                <a class="ml-3" href="https://extranet.interactive-biosoftware.com/alamut-visual-plus_API.html" target="_blank">?</a>
+                <a class="ml-3" href="https://extranet.interactive-biosoftware.com/Alamut_Visual_Plus_API_2.0.0.html" target="_blank">?</a>
```

Comment:
Right, the new API. Does the link work and opens a window correctly?

Reply:
Yep, all good!